website/api/recursion.md

```diff
@@ -0,0 +1,123 @@
+# Recursive Proving
+
+RISC Zero's zkVM uses recursive proving in order to achieve unbounded computation size, constant\* proof size, proof aggregation, and proof composition.
+
+The end-to-end process for proof generation is shown in the following diagram, including links to the associated crate docs:
```

Comment:
It's unclear to me what value the links add. Who is this doc aimed at an what are they interested in figuring out? One thing I don't want is to give developers the impression that they need to be using the methods and types linked here, which should be implementation details for most developers.

Reply:
I'm also concerned that these links will go stale over time.

Reply:
this doc is housed under `reference` and not something that most users would encounter unless they're digging into questions about architecture, or we point them to it directly. 

at this stage, our `reports.risczero.com` page is constructed in a way that is going to invite all sorts of questions about these operations -- e.g. [here](https://discord.com/channels/953703904086994974/1152032592170844180/1247999471758086207). 

i'd like to have an easy-to-link-to & comprehensive answer to these questions so that we don't have to re-explain things a million times.

Reply:
i'll add an :::info block that clarifies that this doc is provided for informational purposes and that users don't need to use these functions directly

Reply:
re: stale links... 

not sure how this is any different than everywhere else on the dev docs site...?
note that there's only really like 5 links in the diagram -- would be pretty straight-forward to ctrl-f and update.

Reply:
i'm open to dropping the links if y'all think that's best but generally don't like having informational dead ends

Reply:
i guess another option would be to drop the item-by-item links and just add a single link to the ApiClient page? 

/shrug

Reply:
I think most users won't be using the `ApiClient` directly, instead they should use the `Prover` trait.

Reply:
> I think most users won't be using the ApiClient directly, instead they should use the Prover trait.

Thanks Frank! That's helpful.